starters/features/bootstrap/src/components/bootstrap/navbar.tsx

```diff
@@ -0,0 +1,47 @@
+import { component$, useVisibleTask$ } from "@builder.io/qwik";
+import { Link } from "@builder.io/qwik-city";
+export const Navbar = component$(() => {
+  useVisibleTask$(async () => {
+    /**
+     * It is required for Javascript functions within Bootstrap components to work.
+     * For example, if we adapt to a mobile screen, so that the toggle menu button works.
+     * Without this it will NOT work (try it if you want)
```

Comment:
(maybe remove this line, as it feels a bit snarky )

Reply:
No problem @mhevery, I'll delete it. If in the future I have to make any indication, I understand that I have to indicate it in the documentation so that it is cleaner. Thank you so much! .